assets/js/modules/analytics/components/setup/SetupFormGA4.js

```diff
@@ -116,15 +125,17 @@ export default function SetupFormGA4() {
 				<GA4PropertySelect />
 			</div>
 
-			{ hasExistingGA4Tag && <SetupUseSnippetSwitch /> }
+			{ hasExistingGA4Tag && <SetupUseSnippetSwitchGA4 /> }
 
 			{ shouldShowAssociatedPropertyNotice && (
 				<GA4PropertyNotice
 					notice={ __(
 						'An associated Universal Analytics property will also be created.',
 						'google-site-kit'
 					) }
-				/>
+				>
+					{ hasExistingUATag && <SetupUseSnippetSwitchUA /> }
+				</GA4PropertyNotice>
```

Comment:
The IB states:
>In assets/js/modules/analytics/components/setup/SetupFormGA4.js:
If the existing GA4 tag is available, render the SetupUseSnippetSwitch component as a children prop within the GA4PropertyNotice component.

However, this is incorrect since the toggle snippet for the GA4 property has already been rendered above. As for the notice, we should render the toggle snippet for the UA property. Moreover, the notice text says, `An associated Universal Analytics property will also be created.`.